app/models/like.rb

```diff
@@ -0,0 +1,10 @@
+class Like < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+
+  after_save :update_comments_counter
```

Comment:
- [ ] Great job on your project ,you have done a great job so far but there are still some isues you still need to address ,please kindly add Rspect unit test that describe the like ,as you can see  acccording to the project requirement its mandatory that  Any custom methods in the models should have a corresponding unit test.